CMakeLists.txt

```diff
@@ -61,6 +61,8 @@ endif()
 if(CMAKE_CXX_COMPILER_ID STREQUAL "Clang" AND ENABLE_LIBCXX)
     set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -lc++ -lc++abi")
     set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -lc++ -lc++abi")
+elseif(CMAKE_CXX_COMPILER_ID STREQUAL "Clang" AND NOT ENABLE_LIBCXX)
+    add_compile_options(-fsized-deallocation)
```

Comment:
Looks like this is required for GCC, too (line 68).  Let's instead put this in an `if()` branch above line 58 and delete line 68.

If we're enabling libc++, then we don't set this flag, and we set a different one; if we are enabling libc++, then we get this flag and not the other.

Reply:
resololved